code/__HELPERS/unsorted.dm

```diff
@@ -1875,6 +1875,21 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 
 #define RANDOM_COLOUR (rgb(rand(0,255),rand(0,255),rand(0,255)))
 
+// This proc returns every player with a client who is not a ghost or a new_player
+/proc/get_living_players(var/exclude_nonhuman = FALSE, var/exclude_offstation = TRUE)
```

Comment:
Procs shouldnt contain var/ in their variables, just for standardization
```suggestion
/proc/get_living_players(exclude_nonhuman = FALSE, exclude_offstation = TRUE)
```

Reply:
pushed it just a second ago, linter already yelled at me :D